packages/jazz-tools/src/tests/syncServer.ts

```diff
@@ -0,0 +1,89 @@
+import { createServer } from "http";
+import { ControlledAgent, LocalNode } from "cojson";
+import { createWebSocketPeer } from "cojson-transport-ws";
+import { WebSocket, WebSocketServer } from "ws";
+import { TestJSCrypto } from "../testing";
+
+export const startSyncServer = async (port?: number) => {
```

Comment:
Added a syncServer in the jazz-tools utils to write the tests for WebSocketPeerWithReconnection more easily.